pyoneer/layers/dropout_test.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+from pyoneer.layers.dropout_impl import ConcreteDropout
+
+
+class DropoutTest(tf.test.TestCase):
+    def test_concrete_dropout(self):
```

Comment:
Can we test the regularization loss, as well? I.e. for different drop out rates, the regularization should change?

Reply:
Yep, this was rushed. It definitely needs more tests.